docs/src/further_topics/ugrid/data_model.rst

```diff
@@ -0,0 +1,361 @@
+by every data point/area along that line.
+
+For example: representing a cubed-sphere using a mesh leads to coordinates and
+connectivities being **~8 times larger than the data itself**, as opposed to a
+small fraction of the data size when dividing a sphere using a structured grid.
+
+This further increases the emphasis on lazy loading and processing of data
+using packages such as Dask.
+
+.. note::
+
+        The large, 1D data arrays associated with meshes are a very different
+        shape to what Iris users and developers are used to. It is suspected
+        that optimal performance will need new chunking strategies, but at time
+        of writing (``Jan 2022``) experience is still limited.
+
+.. todo:
+        Revisit when we have more information.
+
+Spatial operations on mesh data are more complex
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Detail: :doc:`operations`
+
+Indexing a mesh data array cannot be used for:
+
+#. Region selection
+#. Neighbour identification
+
+This is because - unlike with a structured data array - relative position in
+a mesh's 1-dimensional data arrays has no relation to relative position in
+space. We must instead perform specialised operations using the information in
+the connectivities present, or by translating the mesh into a format designed
+for mesh analysis such as VTK.
+
+Such calculations can still be optimised to avoid them slowing workflows, but
+the important take-away here is that **adaptation is needed when working mesh
+data**.
+
+
+How Iris Represents This
+========================
+
+..
+    Include API links to the various classes
+
+    Include Cube/Mesh printout(s)
+
+.. seealso::
+
+        Remember this is a prose summary. Precise documentation is at:
+        :mod:`iris.experimental.ugrid`.
+
+.. note::
+
+        At time of writing (``Jan 2022``), neither 3D meshes nor 3D elements
+        (volumes) are supported.
+
+The Basics
+----------
+The Iris :class:`~iris.cube.Cube` has several new members:
+
+* | :attr:`~iris.cube.Cube.mesh`
+  | The :class:`iris.experimental.ugrid.Mesh` that describes the
+    :class:`~iris.cube.Cube`\'s horizontal geography.
+* | :attr:`~iris.cube.Cube.location`
+  | ``node``/``edge``/``face`` - the mesh element type with which this
+    :class:`~iris.cube.Cube`\'s :attr:`~iris.cube.Cube.data` is associated.
+* | :meth:`~iris.cube.Cube.mesh_dim`
+  | The :class:`~iris.cube.Cube`\'s **unstructured dimension** - the one that
+    indexes over the horizontal :attr:`~iris.cube.Cube.data` positions.
+
+These members will all be ``None`` for a :class:`~iris.cube.Cube` with no
+associated :class:`~iris.experimental.ugrid.Mesh`.
+
+This :class:`~iris.cube.Cube`\'s unstructured dimension has multiple attached
+:class:`iris.experimental.ugrid.MeshCoord`\s (one for each axis e.g.
+``x``/``y``), which can be used to infer the points and bounds of any index on
+the :class:`~iris.cube.Cube`\'s unstructured dimension.
+
+.. todo: Cube printout
```

Comment:
This was the one thing I felt was missing from this page so this will be a great addition

Reply:
3692c1cfa51f15181437b22088f3f223d0037a93